Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -447,7 +448,8 @@ def _create_exposure_filter(exposure_arg):
     # Because the CLI can't accept lists, convert the args to a list if the type is string.
```

Comment:
This comment looks like it was copied from CLI code, which could be a little confusing here.

Reply:
or is this referring to the `demisto-sdk` CLI?

Reply:
i don't see this comment in the codebase for the CLI, so nevermind, you can leave this as is.

Reply:
This is the CLI in Cortex XSOAR, like where you type commands in War Rooms